### PR TITLE
0.2.5 release

### PR DESCRIPTION
**Pull request by freakboy3742**

### CODE REVIEW

File: rubicon/objc/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-__version__ = '0.2.3'
+__version__ = '0.2.5'
```
Comment (dgelessus) [inline rubicon/objc/__init__.py]: What happened to 0.2.4 though? I see it's on https://pypi.python.org/pypi/rubicon-objc, but in the repo it existed for two minutes before being replaced with 0.2.5.

**Comment by dgelessus** [comment]
Also, is there a changelog for rubicon-objc somewhere? If not, I could see if I can make one.

File: rubicon/objc/__init__.py (hunk 1 above)
Comment (freakboy3742) [inline rubicon/objc/__init__.py]: I messed up. I pushed the PyPI version before checking I had all the latest Github updates.

**Comment by freakboy3742** [comment]
There isn't a changelog - but if you are motivated to write one, that would be amazing!

**Pull request merged**